internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1156,7 +1156,8 @@ func (r *ClusterDeploymentsReconciler) syncManifests(ctx context.Context, log lo
 				Content:  swag.String(base64.StdEncoding.EncodeToString([]byte(manifest))),
 				FileName: swag.String(filename),
 				Folder:   swag.String(models.ManifestFolderOpenshift),
-			}})
+			}},
+			false)
```

Comment:
Manifests originating from the configmap are custom user manifests, why don't we consider them custom user manifets?